src/PackageBundler.jl

```diff
@@ -237,7 +237,7 @@ function bundle(
                 end
                 artifact_name = "$name.tar.gz"
                 temp_artifact = joinpath(temp_dir, artifact_name)
-                download_hash = Pkg.Artifacts.archive_artifact(product_hash, temp_artifact)
+                download_hash = _archive_artifact(product_hash, temp_artifact)
```

Comment:
I'd also just be interested in seeing the top-level comparison of `@time Pkg.Artifacts.archive_artifact(product_hash, temp_artifact)` versus `@time _archive_artifact(product_hash, temp_artifact)` on the same platform.

Reply:
For a large bundle we are seeing the following improvements:


Before:
```
381.070133 seconds (428.66 k allocations: 30.277 MiB, 0.15% compilation time)
```

After:
```
12.998432 seconds (160.68 k allocations: 13.061 MiB, 0.19% compilation time)
```

Reply:
That is wild. So it's some problem with `7z`?

Can you print out the hash of the tarball for both methods? I want to make sure we're actually creating the same tarball with both methods.

Reply:
@DilumAluthge ^